pyproject.toml

```diff
@@ -66,7 +66,7 @@ srv = [
     "pymongo[srv]>=4.5,<5",
 ]
 test = [
-    "pytest>=7", "mockupdb", "tornado>=5", "aiohttp<3.8.6", "motor[encryption]"
+    "pytest>=7", "mockupdb", "tornado>=5", "aiohttp>=3.8,!=3.8.6", "motor[encryption]"
```

Comment:
Why did we upgrade to `aiohttp>=3.8`?

Reply:
I now realize this is only the for the test suite, not the motor deps for users. So this is fine.